.github/workflows/deploy.yml

```diff
@@ -24,6 +18,12 @@ jobs:
           path: node_modules
           # Don't share with other workflows because we are using `--production=false`
           key: dev-dependencies-${{ hashFiles('**/yarn.lock') }}
+      - name: Setup fontawesome dependencies
```

Comment:
Does moving it here allow the node modules to stay cached thus speeding things up I'm guessing?

Reply:
Yeah it feels like a better place. Though I'm not sure.